openedx/core/lib/docs/decisions/0002-logging-and-monitoring-expected-errors-removed.rst

```diff
@@ -0,0 +1,44 @@
+Logging and monitoring of expected errors removed
+=================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The setting ``EXPECTED_ERRORS`` was used for both expected and ignored errors in edx-platform.
+
+The functionality supporting expected errors was added:
+
+* Because New Relic didn't yet support this for Python, and
+* Because it was thought that people may want more flexibility for marking errors as expected in addition to just the class name, and
+* The setting provides a place to document why the change is being made.
+
+Updates regarding these three original reasons:
+
+* New Relic now supports expected errors, so the custom functionality is redundant and complicated, and
+* This functionality has yet to be used, so it is also unnecessary, and
+
+Note that the need for custom functionality for ignored errors differs from expected errors in the following ways:
+
+* New Relic still has problems from time to time where it stops ignoring errors that it is supposed to be ignoring, and this redundant functionality is used to catch that issue, and
+* Since New Relic does not capture details of ignored errors, the custom functionality to log these errors can still come in handy if more details are needed.
+
+This context can also be found in `[DEPR] Expected error part of EXPECTED_ERRORS`_.
+
+.. _[DEPR] Expected error part of EXPECTED_ERRORS: https://github.com/openedx/edx-platform/issues/32405
+
+Decision
+________
+
+The custom ignored error functionality proposed in 0001-logging-and-monitoring-ignored-errors.rst will remain in place, but the custom expected error functionality will be removed.
+
+Consequence
+-----------
+
+* The setting ``EXPECTED_ERRORS`` will be renamed to ``IGNORED_ERRORS``, which better matches how it was being used in the first place.
```

Comment:
May want to mention the change from `REASON_EXPECTED` to `REASON_IGNORED` in the values, too.

Reply:
I added more details. I don't want to enumerate everything. Hopefully this is enough, and otherwise, I'll pare it down and make it more generic.